node-api/src/error.rs

```diff
@@ -95,6 +95,9 @@ pub enum RuntimeError {
     /// At least one consumer is remaining so the account cannot be destroyed.
     #[error("At least one consumer is remaining so the account cannot be destroyed.")]
     ConsumerRemaining,
+    /// There are too many consumers so the account cannot be created.
```

Comment:
Seems to be quite redundant. Is it really necessary?

Reply:
I agree with you, I just took it from the original error in substrate.